contrib/awssqs/config/201-clusterrole.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: awssqs-controller
+rules:
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - VerbAll
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - VerbAll
+- apiGroups:
+# TODO on these and the eventing ones that this should be replaced with
+# an aggregated role for watching Addressables.
+  - sources.eventing.knative.dev
+  resources:
+  - awssqssources
+  verbs:
+  - VerbAll
```

Comment:
I don't think you need "create" or "delete" on the objects that you are controlling -- *your* controller should never be creating or deleting these things, that's up to the user or higher-level system. So I'd expect:
```
 - ["get", "list", "watch", "update", "patch"]